app/src/store/voice/accessToken.ts

```diff
@@ -54,14 +59,25 @@ export const getAccessToken = createTypedAsyncThunk<
   }
 
   const tokenResponse = fetchResult.value;
+
+  const tokenTextResult = await settlePromise(tokenResponse.text());
+
+  if (!tokenResponse.ok && tokenTextResult.status === 'fulfilled') {
```

Comment:
I'm not quite seeing this branch as being necessary. This kind of error should be caught by the branch on line `73`, no?

Reply:
Ah, I see now. We want the value of the `text`.

Reply:
Yes I get error:
```
Property 'value' does not exist on type '{ status: "rejected"; reason: unknown; } | { status: "fulfilled"; value: string; }'.
  Property 'value' does not exist on type '{ status: "rejected"; reason: unknown; }'.
```